### DISCUSSION TITLE
Erasing data

### DISCUSSION THREAD

**Discussion by mokamars** [prisma/prisma#14715] [Q&A]
Hi everyone, I'm pretty sure the answer will be easy :) but I can't find any solution (stackoverflow or googling).
I'm currently usine Prisma with a sqlite database. Each time I run the command "npx prisma db push" all my data are erased. And I can't find a soultion to migfrate my production database without resetting the data ?
Do I need to script a dump or something like that ?

Thanks

**Comment by ludralph** [comment]
Hi @mokamars 👋 

`db push` works well if:

You want to quickly prototype and iterate on schema design locally without the need to deploy these changes to other environments such as other developers, or staging and production environments.

In a development environment, use the migrate dev command to generate and apply migrations:

`npx prisma migrate dev`

In production and testing environments, use the migrate deploy command to apply migrations:

`npx prisma migrate deploy`

For more information, please check the Prisma [docs on migrations](https://www.prisma.io/docs/concepts/components/prisma-migrate)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
